docker_plugin/tasks.py

```diff
@@ -267,7 +267,8 @@ def pull(client, arguments):
             .format(arguments, str(e)))
 
     image_id = utils.get_image_id(
-        arguments.get('tag'), image_id, client)
+        arguments['tag'], arguments['repository'], client)
```

Comment:
maybe "arguments.get('tag'), arguments.get('repository')" is better? with defaults? or do you want to fail here if they are missing?